src/applications/claims-status/reducers/serialize.js

```diff
@@ -25,27 +29,49 @@ const getDocsAssociatedWithTrackedItems = docs =>
   docs.filter(isAssociatedWithAnyTrackedItem);
 
 const getDocsAssociatedWithTrackedItem = (item, docs) => {
-  const predicate = isAssociatedWithTrackedItem(item);
-  return docs.filter(predicate);
+  return docs.filter(isAssociatedWithTrackedItem(item));
 };
 
 const associateDocsWithTrackedItems = (items, docs) => {
-  return items.map(item => {
-    const newItem = { ...item };
-    const associatedDocs = getDocsAssociatedWithTrackedItem(newItem, docs);
-    newItem.documents = associatedDocs || [];
-    newItem.date = getTrackedItemDate(item);
-    return newItem;
-  });
```

Comment:
Cleaned this up a little to make it more concise. Notable changes are:
1. I removed the `|| []` for assigning `documents`, since I don't believe we can get here with a falsy return value for `getDocsAssoociatedWithTrackedItems` (since `docs` need to be an array already to run `filter` on them). For additional safety, I went ahead and added a default value (empty array) for both `supportingDocuments` and `trackedItems` at the beginning of the `serializeClaim` function.
2. I moved setting `date` on the tracked item into the transform function, as I don't believe it's related to associating docs with the tracked item.